app/controllers/admin/project_steps_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::ProjectStepsController < Admin::AdminController
+  helper TimeLanguageHelper
+
+  def destroy
+    @step = ProjectStep.find(params[:id])
+    if @step.destroy
+      redirect_to admin_loan_path(@step.project_id), notice: I18n.t(:notice_deleted)
```

Comment:
This notice does not show when the project step is completed. These methods may need to be adjusted.